File: spec/javascripts/task_spec.js
describe("Task", function() {
	var task = new Task({
		description: "Todo App com Backbone.js"
	});

	it("has a description", function() {
		expect(task.get("description")).toEqual("Todo App com Backbone.js");
	});

	it("doesn't start as done", function() {
		expect(task.get("isDone")).toBeFalsy();
	});

	it("can be set as done", function() {
		task.setDone();
		expect(task.get("isDone")).toBeTruthy();
	});
})

File: spec/models/task_spec.rb
require 'spec_helper'

describe Task do
  let(:task) { Task.new name: "Note what to do"}

  it "does not start as done" do
    expect(task.done?).to be_false
  end

  it "can be set as done" do
    task.done!
    expect(task.done?).to be_true
  end

  it "can be set as undone" do
    task.done!
    task.done!
    expect(task.done?).to be_false
  end

end
